src/Symfony/Component/Runtime/ResolverInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Runtime;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ *
+ * @experimental in 5.3
+ */
+interface ResolverInterface
+{
+    /**
+     * @return array{callable():object, mixed[]}
```

Comment:
This is more specific. 
An array is returned, index 0 will always be a callable and index 1 will always me a `mixed[]`

```suggestion
     * @return array{0: callable, 1: mixed[]}
```

Reply:
updated